tests/e2e/provisioning/test/e2e_provisioning_test.go

```diff
@@ -46,4 +50,39 @@ func Test_E2E_Provisioning(t *testing.T) {
 
 	err = ts.dashboardChecker.AssertRedirectedToUAA(dashboardURL)
 	assert.NoError(t, err)
+
+	if ts.IsTestAzureEventHubsEnabled {
+		checkAzureEventHubProperties(ts, t, operationID)
+	}
+}
+
+func checkAzureEventHubProperties(ts *Suite, t *testing.T, operationID string) {
```

Comment:
Does it make sense to check the provisioning state for the resource group and namespace?

Reply:
No, cause this step is done after Kyma runtime is provisioned. So by this time, Azure event hubs should definitely be created. Waiting should not be necessary.

Reply:
Correct, waiting does not make sense at that point. 

What about checking the provisioning state for the resource group and namespace to be successful? To elaborate more, Is the provisioning state for resource group and namespace reflected somehow as part of the Kyma runtime provisioning state? This is not clear to me.